libgit2-sys/build.rs

```diff
@@ -14,7 +14,7 @@ fn main() {
     let try_to_use_system_libgit2 = !vendored && !zlib_ng_compat;
     if try_to_use_system_libgit2 {
         let mut cfg = pkg_config::Config::new();
-        if let Ok(lib) = cfg.range_version("1.4.4".."1.5.0").probe("libgit2") {
```

Comment:
why do we need to support `1.6.0` here already?

Reply:
good point. idk

Reply:
Got to be careful when upgrading libgit2. It doesn't guarantee stable ABI between minor versions.

- https://github.com/rust-lang/git2-rs/pull/817
- https://github.com/libgit2/libgit2/blob/fbea439d4b6fc91c6b619d01b85ab3b7746e4c19/docs/api-stability.md?plain=1#L44-L57

Reply:
@weihanglo The in-repo libgit2 submodule uses a revision from the libgit2 main branch that almost matches libgit2 v1.5.0.